lib/melinis/task.rb

```diff
@@ -47,6 +47,10 @@ def initialize
 
       # 'failures' returns the list of records that are still in failed state and have individual retries left
       @failures = @task.task_failures.to_be_processed(@task.individual_retries_limit)
+
+      # 'For multiple operation on any individual object,
+      # if we want just a single TaskProcessing entry, we need to populate it in overwrite
+      @melinis_task_processing_id = nil
```

Comment:
Call this 'run_id' & expose it with attr_reader at the top given that the other attribute is called current_run